lizmap/modules/lizmap/controllers/features.classic.php

```diff
@@ -170,13 +170,23 @@ public function displayExpression()
 
         $expressions = array_merge($expressions, $additionalFields);
 
+        $sortingField = trim($this->param('sorting_field', ''));
```

Comment:
```suggestion
        $sortingField = trim($this->param('sorting_field', null));
```
As default `virtualFields` `$sortingOrder` value.